content/blog/2019/corda-simple-broadcast-transaction/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Broadcasting a transaction to external organisations
+date: "2019-05-29"
+published: true
+tags: [corda, kotlin, dlt, distributed ledger technology, blockchain]
+cover_image: ./corda-concert.png
+github_url: https://github.com/lankydan/corda-broadcast-transaction
+---
+
+Corda does not globally broadcast transactions between organisations/nodes by default. Privacy and not sharing unnecessary data is a central component of Corda's design. Compared to other DLT (Distributed Ledger Technology) platforms and blockchains, this is indeed a big difference. But, this statement comes with one fallacy. There is a belief that Corda cannot broadcast transactions to external nodes at all. This is simply wrong. It may not be the default behaviour but it is definitely within Corda's capabilities. In this post, I will demonstrate the small amount of code required to send a transaction to any desired node.
+
+That being said, the code included in this post is for a trivial implementation. Moving forward, I aim to write further posts on this subject and move towards more complex and useful implementations for broadcasting transactions.
+
+Before I show you any code. We should have a quick talk about the need for broadcasting transactions to parties not originally involved in a transaction. The most common reason I have come across is the need to meet regulatory requirements. For certain workflows information has to be shared with third-parties. These parties can then validate that nothing dodgy is going on and might also be a link to the _real_ world. I am not going to lie, this isn't exactly my area of expertise. Luckily, if you are reading this because you have concerns about meeting regulatory requirements. Then you probably know better than me as to why you need the ability to broadcast transactions.
```

Comment:
"the need for broadcasting **data**" ?
you can also broadcast reference data to all the nodes that need it, etc.